ports/stm32/boards/NUCLEO_WB55/rfcore_make_firmware.py

```diff
@@ -0,0 +1,79 @@
+#!/usr/bin/env python3
```

Comment:
should this file be called `rfcore_makefirmware.py`, which matches things like `makemanifest.py`?